test/unit/requester-core.test.js

```diff
@@ -287,6 +287,20 @@ describe('requester util', function () {
             expect(requesterCore.getRequestBody(request)).to.have.property('body');
         });
 
+        it('should handle disabled request bodies correctly ', function () {
+            var request = new sdk.Request({
+                url: 'postman-echo.com/post',
+                method: 'POST',
+                body: {
+                    disabled: true,
+                    mode: 'raw',
+                    raw: '{"beta":"bar"}'
+                }
+            });
+
+            expect(requesterCore.getRequestBody(request)).to.be.undefined;
```

Comment:
Is this also the behaviour when no body exists?

Reply:
Yes